doc/Changelog.md

```diff
@@ -1,5 +1,9 @@
 # Changelog {#Changelog}
 
+* [19](https://github.com/HBPVIS/ZeroBuf/pull/19):
```

Comment:
Common practice so far: Each change/commit/feature is a separate entry in the change log. The fact that they were merged in a single PR is due to speed( coder ) > speed( reviewers )